jetbrains-core/it/software/aws/toolkits/jetbrains/services/clouddebug/CloudDebugTestCase.kt

```diff
@@ -87,15 +88,36 @@ abstract class CloudDebugTestCase(private val taskDefName: String) {
 
     @After
     open fun tearDown() {
+        try {
+            deinstrumentService()
+        } finally {
+            // If deinstrumenting fails, or initialization doesn't work properly, we still want to try to delete the services, so kick that off
+            try {
```

Comment:
if you dont care about results, you can wrap in `runCatching {}`

Reply:
I read this as `runCaching` which really confused me. Good point, changed